plasma_framework/contracts/src/utils/RLP.sol

```diff
@@ -55,9 +55,9 @@ library RLP {
     }
 
     /**
-     * @dev Creates an RLPItem from an array of RLP encoded bytes.
-     * @param self The RLP encoded bytes.
-     * @param strict Will throw if the data is not RLP encoded.
+     * @dev Creates an RLPItem from an array of RLP-encoded bytes
+     * @param self The RLP-encoded bytes
+     * @param strict Triggered when data is not RLP-encoded
```

Comment:
not sure but it seems both old and new version is misleading. Shouldn't this be:

`If True, will throw if the data is not RLP-encoded`

But this is just an educated guess, can someone confirm?